flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CheckpointStatsTracker.java

```diff
@@ -183,23 +166,27 @@ public CheckpointStatsSnapshot createSnapshot() {
     /**
      * Creates a new pending checkpoint tracker.
      *
+     * @param tasksToAck Tasks to acknowledge in this checkpoint.
      * @param checkpointId ID of the checkpoint.
      * @param triggerTimestamp Trigger timestamp of the checkpoint.
      * @param props The checkpoint properties.
      * @return Tracker for statistics gathering.
      */
     PendingCheckpointStats reportPendingCheckpoint(
-            long checkpointId, long triggerTimestamp, CheckpointProperties props) {
+            List<ExecutionVertex> tasksToAck,
```

Comment:
Should still be `tasksToWaitFor`

Reply:
The method has been evolved, thus we do not need to modify this for now.